Update BoltCard Extension README (#1241)
lnbits/lnbits: lnbits/extensions/boltcards/README.md

    - Or you can Click the "KEYS / AUTH LINK" button to copy the auth URL to the clipboard. Then paste it into the Android app (Create Bolt Card -> PASTE AUTH URL).
- Click WRITE CARD NOW and approach the NFC card to set it up. DO NOT REMOVE THE CARD PREMATURELY! 

## Rewriting / Erasing the card - Boltcard NFC Card Creator

> It is possible not only to reset the keys but also to disable the SUN function and completely erasing the card so it can be used again as a static tag or set up as a new Bolt Card. 

Reviewer: ```suggestion
It is possible not only to reset the keys but also to disable the SUN function and completely erase the card so it can be used again as a static tag or set up as a new Bolt Card. 
```

IMPORTANT: 
* It is immanent that you have access to your old keys so do not erase this card in LNbits before you copied those strings!
* If you tried to write to them and failed you will need the same amount of positive writing sessions to unlock the card.

- in the BoltCard-Extension click the QR code button next to your old card and copy Key0
- in the BoltApp click Advanced - Reset keys and paste the Key0 into the first field named Key0
- repeat with Key1/Key2/Key3/Key0
- when done pasting all 4 keys scan your card with the BoltApp
- Thats it 🥳

> - Now if there is all success the card can be safely deleted from LNbits (but keep the keys backuped anyway; batter safe than brick).

Reviewer: ```suggestion
- If everything was successful the card can be safely deleted from LNbits (but keep the keys backed up anyway; batter safe than brick).
```
